client/src/components/rules.jsx

```diff
@@ -153,7 +153,7 @@ const Rules = ({ list }) => {
           </InfoDisplay>
         </Col>
       </Row>
-    </Container>
+    </>
```

Comment:
Probably same here.

Reply:
Thanks @slassen. I took those containers out and replaced them with fragments as those pages are now inside the Container on page.jsx.